script.js

```diff
@@ -196,4 +196,25 @@ form.addEventListener('submit', (e) => {
     e.preventDefault();
     errorMsg.classList.add('display-content');
   }
-});
\ No newline at end of file
+});
+
+// form localStorage Setting
+
+document.getElementById('submit').addEventListener('click', () => {
+  const formDataObj = {
+    name: document.getElementById('formName').value,
+    email: document.getElementById('email').value,
+    message: document.getElementById('comment').value,
+  };
+  const formObj = JSON.stringify(formDataObj);
+  localStorage.setItem('formData', formObj);
+});
+document.getElementById('reset').addEventListener('click', () => {
+  localStorage.removeItem('formData');
+});
```

Comment:
- [ ] After filling out the form, and not submitting it, the entered values should be saved in the browser. Currently, the data is only saved to `localStorage` upon clicking the submit button. Kindly fix that.

Reply:
solved thanks